src/popup/views/GrantAccess/index.tsx

```diff
@@ -1,8 +1,43 @@
 import React from "react";
+import styled from "styled-components";
 import { useLocation } from "react-router-dom";
+
 import { rejectAccess, grantAccess } from "api/internal";
 
-const GrantAccess = () => {
+import { COLOR_PALETTE, FONT_WEIGHT } from "popup/styles";
+import { Button } from "popup/basics";
+
+const GrantAccessEl = styled.div`
+  padding: 2.25rem 2.5rem;
+`;
+const H1 = styled.h1`
+  color: ${COLOR_PALETTE.primary}};
+  font-weight: ${FONT_WEIGHT.light};
+  margin: 1rem 0 0.75rem;
+`;
+const H3 = styled.h3`
+  font-weight: ${FONT_WEIGHT.bold};
+  font-size: 0.95rem;
+  letter-spacing: 0.1px;
+  color: ${COLOR_PALETTE.primary}};
+`;
```

Comment:
some good feedback I got from Morley was to not name vars as their html tag, like `H1` or `H3`, but to name them something like `Header` and `Subheader` as the underlying html tag can change

Reply:
Updated it!